server/services/reportStatusChecker.test.js

```diff
@@ -45,6 +45,7 @@ describe('statusCheck', () => {
       healthcareInvolved: true,
       prisonerRelocation: 'OWN_CELL',
       relocationCompliancy: false,
+      relocationType: 'relocation_to_vehicle',
```

Comment:
doesn't really matter but this should techincally be `VEHICLE`, would be generally good to refer to the constants you set up in Types. `Types.VEHICLE.value` or whatever

Reply:
i see